README.md

```diff
@@ -19,6 +19,38 @@ urlwatch is intended to help you watch changes in webpages and get notified
 changes. The change notification will include the URL that has changed and
 a unified diff of what has changed.
 
-* Documentation: https://urlwatch.readthedocs.io/
-* Website: https://thp.io/2008/urlwatch/
-* E-Mail: m@thp.io
+## Features
+
+- Monitor webpages, headless browser sessions, or shell commands with unified diffs
+- Chain filters to extract, clean up, and normalize content before diffing
+- Send notifications via console, e-mail, and many third-party services
+- YAML-based configuration that is easy to version and schedule via cron
+
+## Requirements
+
+- Python 3.9 or newer
+
+## Installation
+
+Install or upgrade with a single command:
+
+```bash
+python -m pip install --upgrade urlwatch
+```
+
+For more installation details, see the source documentation:
+[docs/source/installation.rst](docs/source/installation.rst).
+
+## Quick start
+
+1. Run `urlwatch` once to initialize data
+2. Edit jobs and filters: `urlwatch --edit` (writes `urls.yaml`)
+3. Edit settings and reporters: `urlwatch --edit-config` (writes `urlwatch.yaml`)
+4. Add `urlwatch` to cron (e.g., `crontab -e`) to run on a schedule
+
+## Documentation and links
+
+- Documentation: https://urlwatch.readthedocs.io/
+- Documentation (source): [docs/source/index.rst](docs/source/index.rst)
+- Website: https://thp.io/2008/urlwatch/
+- E-Mail: m@thp.io
```

Comment:
The changes in README.md should be reverted, the `* Documentation: https://urlwatch.readthedocs.io/` link already provides quick access to the latest docs, which contain that information.